Update Jump Feeds

davevsdave opened the pull request.

davevsdave committed:
Update Jump Feeds

Update feed url scheme for: Austin, Denver, Los Angeles, New York City, Denver, Sacramento, San Diego, Santa Cruz, Seattle, and San Francisco.
New Feed Added: Tampa

davevsdave committed:
Fix typo: "CA" to "FL" for Tampa Jump Feed

jcn reviewed (changes requested):
Thanks for this PR! Can you update the System Names to match what's listed in the feeds themselves (which will also help differentiate between the scooter and bike feeds). Thanks.

davevsdave committed:
Update system names based on system_information.json

A bit less clean organizationally in the list using this name format, but accurate to the feed names.  Additionally, removed Providence feed as they have terminated that program and Chicago's system feeds are returning 404 so I simply used the same naming format from other feeds.

davevsdave commented:
These changes should be addressed. Let me know if you need something additional.

jcn commented on systems.csv:
```diff
@@ -204,20 +204,25 @@ US,Indego,"Philadelphia, PA",bcycle_indego,https://www.rideindego.com,https://gb
-US,JUMP SF,"San Francisco, CA",jump_sf,https://jump.com/,https://sf.jumpbikes.com/opendata/gbfs.json
+US,Austin Bike JUMP System,"Austin, TX",austin_bike_jump_system,https://jump.com/,https://gbfs.uber.com/v1/atxb/gbfs.json
+US,Austin Scooter JUMP System,"Austin, TX",austin_scooter_jump_system,https://jump.com/,https://gbfs.uber.com/v1/atxs/gbfs.json
+US,Chicago Bike JUMP System,"Chicago, IL",jump_chicago,https://jump.com/,https://chi.jumpbikes.com/opendata/gbfs.json
```
This does not appear to be an active feed (all subpages 404).

davevsdave replied:
Yea, so Chicago revoked their temporary permit about a month ago for all scooters/bikes. I think they are going to renew in the spring. Up to you all if you want to remove and then add back in, happy to update the commit by removing that URL. LMK if that's what you'd like.

jcn replied:
@davevsdave Sorry, let's remove it and keep the file full of valid feeds for now. Thanks for taking the time with these edits.

davevsdave replied:
All good - The Chicago jump feeds have been removed and I also popped in the BOLT LA feed.

jcn reviewed (changes requested):
Thanks for the update. One last comment - looks like the Chicago Bike JUMP url is 404ing for all internal pages. Is there a better URL for that one?

davevsdave committed:
Remove Chicago Jump Feeds

davevsdave committed:
Add Bolt Los Angeles Feed
